code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -192,7 +192,7 @@
 	genes = list(/datum/plant_gene/trait/plant_type/fungal_metabolism)
 	growing_icon = 'icons/obj/hydroponics/growing_mushrooms.dmi'
 	reagents_add = list(/datum/reagent/consumable/nutriment = 0.1)
-	mutatelist = list(/obj/item/seeds/chanterelle/jupitercup)
+	mutatelist = list(/obj/item/seeds/chanter/jupitercup)
```

Comment:
why repath this?

Reply:
To make jupiter cups a subtype of chanterelles, much like every other plant and it's mutation(s)

Reply:
no, why did you change chantrelle to chanter. There is no reason to do that

Reply:
Oh I see the existing path is chanter. that is dumb